frontend/src/pages/AccountSend.vue

```diff
@@ -872,14 +866,13 @@ function useSendForm() {
     const tokenToUse = tokenInput || token.value;
     if (!tokenToUse) return tc('Send.select-a-token');
 
-    const { address: tokenAddress, decimals } = tokenToUse;
+    const { address: tokenAddress } = tokenToUse;
 
     // Get total batch send amount for the token
-    const totalBatchSendAmount = summaryAmount.value.get(tokenToUse)?.replace(/,/g, '') || '0';
+    const totalBatchSendAmount = summaryAmount.value.get(tokenToUse) || BigNumber.from(0);
     // Check totalBatchSendAmount is defined and is greater than zero.
-    const totalAmount = parseUnits(totalBatchSendAmount, decimals);
-    if (totalBatchSendAmount && totalAmount.gt(0)) {
-      if (totalAmount.gt(balances.value[tokenAddress])) return `${tc('Send.total-amount-exceeds-balance')}`;
+    if (totalBatchSendAmount && totalBatchSendAmount.gt(0)) {
+      if (totalBatchSendAmount.gt(balances.value[tokenAddress])) return `${tc('Send.total-amount-exceeds-balance')}`;
     }
```

Comment:
This can also be cleaned up:
- `totalBatchSendAmount` is a `BigNumber` so the `if (totalBatchSendAmount)` portion always evaluates to true
- if `totalBatchSendAmount.gt(balances.value[tokenAddress])` is true then `totalBatchSendAmount.gt(0)` must be true, so the latter check is redundant

```suggestion
if (totalBatchSendAmount.gt(balances.value[tokenAddress])) return `${tc('Send.total-amount-exceeds-balance')}`;
```